backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/request/process/WEBUI_CreateRequest.java

```diff
@@ -63,76 +63,76 @@ public WEBUI_CreateRequest()
 	protected String doIt() throws Exception
 	{
 		final String tableName = getTableName();
+		final I_R_Request request;
 		if (I_C_BPartner.Table_Name.equals(tableName))
 		{
 			final I_C_BPartner bPartner = bPartnerDAO.getById(getProcessInfo().getRecord_ID());
-			createRequestFromBPartner(bPartner);
+			request = createRequestFromBPartner(bPartner);
 		}
 		else if (I_M_InOut.Table_Name.equals(tableName))
 		{
 			final I_M_InOut shipment = inOutDAO.getById(InOutId.ofRepoId(getProcessInfo().getRecord_ID()));
-			createRequestFromShipment(shipment);
+			request = createRequestFromShipment(shipment);
+		}
+		else if (I_AD_User.Table_Name.equals(tableName))
+		{
+			final I_AD_User user = userDAO.getById(UserId.ofRepoId(getProcessInfo().getRecord_ID()));
+			request = createRequestFromUser(user);
 		}
 		else
 		{
 			throw new IllegalStateException("Not supported: " + tableName);
 		}
+		save(request);
+		getResult().setRecordToOpen(TableRecordReference.of(request), requestWindowId.get().getRepoId(), OpenTarget.SingleDocumentModal);
```

Comment:
don't blindly assume that `requestWindowId` is present. 
Suggestion: implement `IProcessPrecondition`, then check that in `checkPreconditionsApplicable` if the window is not available, then log a message and and use `rocessPreconditionsResolution.rejectWithInternalReason(..)`